metacat-common-server/src/main/java/com/netflix/metacat/common/server/connectors/model/BaseInfo.java

```diff
@@ -15,20 +15,18 @@
 
 import com.netflix.metacat.common.QualifiedName;
 import lombok.AllArgsConstructor;
-import lombok.Getter;
+import lombok.Data;
 import lombok.NoArgsConstructor;
-import lombok.Setter;
 
 import java.util.Map;
 
 /**
  * Base class for catalog resources.
  * @author amajumdar
  */
-@Getter
-@Setter
-@AllArgsConstructor
+@Data
```

Comment:
Using `@Data` is a bad idea for these type classes as they're meant to transfer data and should be immutable. `@Data` includes setters. All fields should be private final.